sdk/src/TrancheData.ts

```diff
@@ -0,0 +1,30 @@
+import { ReserveFairValue } from "./ReserveFairValue";
+import { TrancheFairValue } from "./TrancheFairValue";
+
+export class TrancheData {
+    depositedQuantity: number[];
+    feeToCollectQuantity: number;
+    reserveFairValue: ReserveFairValue;
+    trancheFairValue: TrancheFairValue;
+    haltFlags: number;
```

Comment:
can we have like an enum for `haltFlags` and `ownerRestrictedIx`? here you can find some refs: https://patrickdesjardins.com/blog/how-to-set-and-read-bitwise-enum-values-in-typescript

Reply:
Created enums for them